test/fixtures/go-binaries/source/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# This shell script can be used to generate the binaries that are located in the parent folder.
+# Simply call this script with no arguments. Needs Docker to be installed.
+# The binaries here are used for the symbol parser and the Go version extraction test.
+
+set -ex
+
+versions="1.13.15 1.16.15 1.18.5 1.19.0"
+# let the user overwrite the versions to build.
+if [ "$#" -gt 0 ]; then
+    versions="$@"
+fi;
+
+for version in $versions; do 
+    # the "latest" builds need an "alpine" tag (there's no "latest-alpine" tag).
+    if [ "$version" = "latest" ]; then
```

Comment:
shouldn't this condition be `==`?

Reply:
TIL: `==` is bash, `=` is POSIX shell. 

This is a shell-script, not a bash-script, so no :-)

Reply:
I did not know that :)